.github/actions/generate-native-package/native.sh

```diff
@@ -11,20 +20,27 @@ git config --global user.email '<>'
 # Checkout thrift files
 git clone https://github.com/guardian/bridget.git
 
-# Set tag from mobile-apps-thrift
 cd bridget
+git checkout $GITHUB_SHA
 CURRENT_VERSION="$(git describe --tags --abbrev=0)"
 cd ../
 
 # Add version const to thrift file
 echo "" >> bridget/thrift/native.thrift
 echo "const string BRIDGET_VERSION = \"$CURRENT_VERSION\"" >>  bridget/thrift/native.thrift
 
+echo "Releasing $RELEASE_TYPE to platform $PLATFORM with version $CURRENT_VERSION"
```

Comment:
So that reads ”Releasing release to…” and “Releasing prerelease to…”. Would “Publishing” be any clearer?

```suggestion
echo "Publishing $RELEASE_TYPE to platform $PLATFORM with version $CURRENT_VERSION"
```

Reply:
I like that suggestion! Thanks

Reply:
I noticed that’s the word used in `gen-typescript.sh`!